x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -937,7 +941,9 @@ export class TaskRunner<
         return { interval: retryInterval };
       }),
       monitoring: this.ruleMonitoring.getMonitoring(),
-      hasError: isErr(schedule),
+      ...(isErr(schedule)
```

Comment:
not your fault but it's super confusing that this variable is just called `schedule` when it's actually "schedule or whatever error running the rule throws" 🙈

Reply:
I'll refactor this `returning always success` part soon (before or during on-week)
Will return the error as error :) and let TM to handle it.
So there will be no taskRunError in the return.